packages/xdl/src/User.ts

```diff
@@ -84,6 +85,10 @@ export class UserManagerInstance {
   _getSessionLock = new Semaphore();
   _interactiveAuthenticationCallbackAsync?: () => Promise<User>;
 
+  constructor() {
+    this.getCachedUserDataAsync = this.getCachedUserDataAsync.bind(this);
```

Comment:
This was required for the `profileMethod` function right? Since it breaks the binding context. Might be worth the check out other/better solutions.

Reply:
I will just remove it since it may cause issues.  The profiling comes out to <1ms so I don't think it needs to be in there.